add dialog

```diff
@@ -0,0 +1 @@
+
```

Comment:
Can you delete this file?

Reply:
Yes, i want it to delete too but don't know how to do that.